hackathon_site/hardware/test_api.py

```diff
@@ -136,6 +136,41 @@ def test_category_get_success(self):
 
         self.assertEqual(expected_response, data["results"][0])
 
+    def test_hardware_quantity_success(self):
+        self._login()
+
+        self.category = Category.objects.create(name="Microcontrollers", max_per_team=4)
+        self.hardware1 = Hardware.objects.create(name="Arduino", quantity_available=2)
```

Comment:
Perhaps add one of these to the `self.category` created in the setUp method as well, and expect the unique counts to be [1, 2]. That way we also cover behaviour when a hardware is in multiple categories, which is possible!